ExtraDry/ExtraDry.Blazor/Components/Dry/Forms/DryInputDateTime.razor

```diff
@@ -0,0 +1,4 @@
+﻿@namespace ExtraDry.Blazor.Forms
+@typeparam T
+
+<input class="@CssClasses" title="@Property?.FieldCaption" type="@DisplayMode" value="@Value" readonly="@ReadOnly" @onchange="HandleChange" @attributes="@UnmatchedAttributes" />
```

Comment:
One of the problems with styling that I've gone around with David is that of the affordance.  The calendar doesn't look too bad on Chrome, but when using input type=search, their magnifying glass is horrible. Given the discussion around currency, should we go ahead and wrap this in a div with an icon and an affordance?  Then, hide the browser calendar affordance like you did with the up/down arrows?

Reply:
Given that there's going to be additional work around a new attribute to pass the icon and affordance through which is needed for the numeric input, do you want me to merge the numeric work into this PR for one super-PR or add the affordance and icon for this input into that piece of work.